e2e/posts-preview.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { test, expect } from "@playwright/test";
+import path from 'path';
+
+import { deletePost, getPostIdInURL, createPostWithFeatureImage } from "./helpers/post";
+
+test.describe('feature image', () => {
+  let postIdsToDelete: string[] = [];
+
+  test.afterAll(async ({ request }) => {
+    // delete all posts created in the tests
+    for (const postId of postIdsToDelete) {
+      await deletePost(request, postId);
+    }
+  })
+
+  test('feature image should be visible in preview', async ({ page, request }) => {
+    // Prepare existing post that has a feature image
+    const postId = await createPostWithFeatureImage(page, request);
+    if (postId) {
+      // Store the post id to delete after the test
+      postIdsToDelete.push(postId);
+    }
+
+    // Open a post
+    await page.goto(`/posts/${postId}`);
+
+    // Open the drawer
+    const drawerButton = page.getByTestId('open-post-drawer');
+    await drawerButton.click();
+
+    // Open the preview
+    const [newPage] = await Promise.all([
+      page.waitForEvent('popup'),
+      page.getByRole('button', { name: 'Preview' }).click(),
+    ]);
+
+    // Wait for the new page to load
+    await newPage.locator('[data-testid="feature-image-preview"]').waitFor();
+
+    // Check that saved feature image is visible
+    expect(newPage.getByTestId('feature-image-preview')).toBeVisible();
+  });
+
+  test('preview should open without feature image', async ({ page, request }) => {
```

Comment:
```suggestion
  test('preview should open without feature image', async ({ page }) => {
```
`request` is unused.